nvflare/private/fed/server/server_engine.py

```diff
@@ -779,6 +780,12 @@ def start_client_job(self, job_id, client_sites):
     def stop_all_jobs(self):
         fl_ctx = self.new_context()
         self.job_runner.stop_all_runs(fl_ctx)
+        self.job_runner.stop()
+
+    def pause_server_jobs(self):
+        running_jobs = list(self.run_processes.keys())
+        for job_id in running_jobs:
+            self.abort_app_on_server(job_id)
```

Comment:
This will cause the JobRunner to stop the job run and update the job status from the Cold server. Need to add the following to remove the job from the JobRunner running_jobs:

            with self.lock:
                del self.job_runner.running_jobs[job_id]